pynxtools/dataconverter/convert.py

```diff
@@ -351,6 +344,7 @@ def convert_cli(
     mapping: str,
 ):
     """The CLI entrypoint for the convert function"""
+    logger.addHandler(logging.StreamHandler())
```

Comment:
Is it needed to set this manually? I always thought logging always sets this as standard

Reply:
It seems we do need to add this for levels below warn. Logging.warn and above have a fallback handler if I remember correctly. 

We can also do logging.basicConfig. That just adds this as part of the config setting routine.

Reply:
We want to go at least to `logging.INFO`, even if we don't use it, right? Then this solution works. 

I thought `logging.basicConfig` [sets the log level for the root logger](https://docs.python.org/3/library/logging.html#logging.basicConfig), would that not effect the NOMAD logging? Maybe I misunderstand..

Reply:
For nomad it should be fine as we should never end up in this part of the code anyways. Generally, I'm just more used to seeing this being set via `basicConfig` but I think just adding the streamHandler is also fine and might be safer in case this function is execution by nomad for whatever reason.

So just go ahead :)

Reply:
Alright, thanks for explaining 👍